prototypes/form-wizard/confirmation.html

```diff
@@ -1,5 +1,7 @@
 ---
 layout: page
+permalink: /confirmation/
```

Comment:
FYI, I had removed these to keep them out of the nav bar, where the "Thank you for your submission" shows up automatically.